src/components/OldUiContentLinkBox/OldUiContentLinkBox.tsx

```diff
@@ -0,0 +1,32 @@
+import { EmptyState, EmptyStateBody, Icon } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import { ContentBox } from 'components/ContentBox/ContentBox';
+
+interface IOldUiContentLinkBoxProps {
+  contentTitle: string;
+  route: string;
+}
+
+export const OldUiContentLinkBox = ({ contentTitle, route }: IOldUiContentLinkBoxProps) => {
+  const baseUrl = process.env.REACT_APP_PNC_OLD_UI_WEB || '';
```

Comment:
@patrikk0123 @jomrazek can you guys agree on updating our Jenkins environment? Possible using primary addresses directly could be sufficient, but it can be less flexible. I will leave it up to you.

Reply:
I would avoid hardcoding the URL.

Reply:
I meant using `getPncUrl()` as it's currently pointing to the old UI, this should be basically true until all bifrost stuff is fully implemented on new UI. But using env is completely right and more flexible.

Reply:
That I did not realize. But, I think, semantically, my way is more correct (let's say the fact that REST and old UI share URL is 'coincidence').

Also, both stage and master dev CD scripts now define REACT_APP_PNC_OLD_UI_WEB environment var.